contracts/src/libraries/FROST.sol

```diff
@@ -93,14 +168,26 @@ library FROST {
         return _h2(abi.encodePacked(rv, rx, yv, yx, message));
     }
 
-    /// @notice Verifies a FROST signature.
+    /**
+     * @notice Verifies a FROST group signature.
+     * @param y The group public key.
+     * @param signature The FROST signature to verify.
+     * @param message The signed message.
+     */
     function verify(Secp256k1.Point memory y, Signature memory signature, bytes32 message) internal view {
         require(signature.z < Secp256k1.N, InvalidScalar());
         uint256 c = challenge(signature.r, y, message);
         Secp256k1.mulmuladd(signature.z, c, y, signature.r);
     }
 
-    /// @notice Verifies a FROST signature share.
+    /**
+     * @notice Verifies a FROST signature share.
+     * @param group The group public key.
+     * @param r The group commitment point.
+     * @param participant The participant public key.
```

Comment:
this is called a verification share in FROST, right?

See https://frost.zfnd.org/frost.html#key-generation